file_browser/src/file_browser/local/partials/1-elements/_buttons.scss

```diff
@@ -0,0 +1,59 @@
+[class*=regular-button] {
+    color: var(--white-color);
+    background-color: var(--color1-normal);
+    font-family: $font;
+    border: none;
+    border-radius: 5px;
+    flex: 1;
+    margin: 15px;
```

Comment:
The flex and margin properties don't belong in a class that defines how the element looks.
They belong in a layout class that you define per-component or in a global `_layouts.scss` file.
Remember that you can give an element as many classes as you want, so there's absolutely no need for the global class to define the layout properties.

**What are layout properties?**
Anything that defines how the element is positioned relatively to its parent. 
`flex`, `left`, `top`, `margin`, `align/justify-self` etc...